### DISCUSSION TITLE
Change argocd health check for a defined resource

### DISCUSSION THREAD

**Discussion by DipayanP007** [argoproj/argo-cd#8559] [Q&A] [answered]
Hey, we have switched to argo-cd recently and we r using apps-of-apps orchestration. But the problem is we have been using traefik ingress and it does not provide us the external ip when the ingress resource is created. And the argocd health-check checks for ip or hostname in `.status.loadBalancer.ingress` . So is there any way we can configure argocd to show the ingress health or completely skip the health check for ingress?

**Comment by RomanBats** [comment, marked as answer]
Well, I guess it would be better to configure your ingress to reflect your environment. Maybe traefik ingress could be made private (to make it not to request external IP) or not be installed at all.
But if it is not possible, you could try writing [Custom Health Checks](https://argo-cd.readthedocs.io/en/stable/operator-manual/health/#custom-health-checks).

**Comment by DipayanP007** [comment]
Thank you @RomanBats . I overwrote the ingress health check to skip checking the ingress and it worked.

**Reply by dignajar** [reply]
Hey, can you share your custom health check for the ingress? thanks
